core/src/network.rs

```diff
@@ -652,7 +659,7 @@ impl NetworkConfig {
     /// When the buffer is full, the background tasks of all connections will stall.
     /// In this way, the consumers of network events exert back-pressure on
     /// the network connection I/O.
-    pub fn set_connection_event_buffer_size(&mut self, n: usize) -> &mut Self {
+    pub fn set_poll_event_buffer_size(&mut self, n: usize) -> &mut Self {
```

Comment:
What motivated this change? Address-change events still count as connection events to me, even if they're not emitted directly by a `ConnectionHandler`. `poll` seems a bit more ambiguous, but if you think its an improvement, why not. But maybe it would make sense to update the doc-comments then to reflect the change in terminology (and clarify what a "poll event" refers to).

Reply:
Err... that wasn't intended and is an accident.